### PR TITLE
Make branch kg-tsv run on Sterling

### PR DESCRIPTION
**Pull request by gaurav**
This PR collects some changes needed to make kg-tsv run on Sterling:
- Increased ephemeral space to 10G so that we don't run out of it immediately.
- Moved Coursier cache into /workspace so that we don't use up our ephemeral space.
- Reduced build-cam-database-storage to 100G, since we only needed ~30G for the current build.
- Updated renciorg/cam-pipeline-tools from v1.4 to latest.
- scala-cli required some Maven packages be upgraded.
- Commented out http://purl.obolibrary.org/obo/ncbitaxon/subsets/taxslim.owl in favor of http://purl.obolibrary.org/obo/ncbitaxon/subsets/taxslim-disjoint-over-in-taxon.owl (see #99 for details)

Also some general updates:
- Updated Biolink model to v3.4.3.

With these settings, I was able to set up a `debug-cam-database` pod on Sterling that I could run `make kg.tsv` in to obtain a kg.tsv file that consisted of 1,605,400 triples.

WIP: should be merged after PR https://github.com/ExposuresProvider/cam-pipeline/pull/97 and once https://github.com/obophenotype/ncbitaxon/issues/76 has been fixed.

### CODE REVIEW

File: ontologies.ofn [hunk 1]
```diff
@@ -20,7 +20,9 @@ Import(<http://purl.obolibrary.org/obo/eco/eco-base.owl>)
 Import(<http://purl.obolibrary.org/obo/uberon/bridge/uberon-bridge-to-caro.owl>)
 Import(<http://purl.obolibrary.org/obo/uberon/bridge/cl-bridge-to-caro.owl>)
 Import(<http://purl.obolibrary.org/obo/go/extensions/go-bfo-bridge.owl>)
-Import(<http://purl.obolibrary.org/obo/ncbitaxon/subsets/taxslim.owl>)
+# Commented out on 2023-jun-19 by Gaurav: appears to be redundant with
+# http://purl.obolibrary.org/obo/ncbitaxon/subsets/taxslim-disjoint-over-in-taxon.owl
+# Import(<http://purl.obolibrary.org/obo/ncbitaxon/subsets/taxslim.owl>)
```
Comment (balhoff) [inline ontologies.ofn:25]: This can be reverted—NCBITaxon is fixed.

**Review by balhoff — CHANGES_REQUESTED** [review]

File: kubernetes/build-cam-database-storage.yaml [hunk 2]
```diff
@@ -10,5 +10,5 @@ spec:
   - ReadWriteMany
   resources:
     requests:
-      storage: 1000G
+      storage: 100G
```
[the remark this answers is not in the document]
Comment (gaurav) [reply kubernetes/build-cam-database-storage.yaml]: I'll increase it to 500Gi -- I think the 1000G was designed to hold 3-4 versions of the old cam-pipeline (current, in-process, and then the biolink-3.0 version I was working on). Done in 291012b.

File: ontologies.ofn (hunk 1 above)
Comment (gaurav) [reply ontologies.ofn]: Reverted in ed85f7b.

**Comment by gaurav** [comment]
I've confirmed that the two changes worked, and produces a kg.tsv with 1,605,400 rows in it.

**Review by balhoff — APPROVED** [review]

**Pull request merged**